hrm/department_app/models/department_models.py

```diff
@@ -0,0 +1,29 @@
+"""Django models"""
+from django.db import models
+
+class Department(models.Model):
+    """Department"""
+
+    name = models.CharField('Department',max_length=50,primary_key=True)
+
+    Active= "Active"
+
+    Inactive = "Inactive"
+
+    STATUS_CHOICES = (
+        (Inactive, "Inactive"),
+        (Active, "Active"),
+    )
+
+    status = models.CharField(max_length=10,
```

Comment:
Why 10? The maximum value of the field is Inactive and consists of 8 characters.